HACKTOBERFEST.md

```diff
@@ -0,0 +1,57 @@
+# Appwrite hacktoberfest contribution
+
+Welcome to the Appwrite Hacktoberfest contribution! We appreciate your interest in contributing to our open-source project. Please read this carefully to understand how to get started and make your contributions count.
+
+## Familiarize with the project
+
+- Before you start contributing, it's essential to get familiarized with our project. You can do that by following the [main README](https://github.com/appwrite/appwrite/blob/main/README.md).
+
+## Code of Conduct
+
+- Please ensure that you follow our [Code of Conduct](https://github.com/appwrite/awesome-appwrite/blob/master/CODE_OF_CONDUCT.md) at all times. We aim to maintain a respectful and inclusive community.
+
+### Contributing Guide
+Please review our [Contributing Guide](https://github.com/appwrite/appwrite/blob/main/CONTRIBUTING.md) before contributing. We strive to maintain a friendly and inclusive community for everyone.
```

Comment:
> We strive to maintain a friendly and inclusive community for everyone.

The contributing guide is more about contributing by giving architecture information and instructions on how to update code, run tests, submit a PR, etc, rather than being inclusive.